srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -171,21 +174,21 @@ private void parseRecentVitalResults(final Patient patient, final List<String> v
     {
     	final SimpleDateFormat dateFormat = new SimpleDateFormat("(" + VISTA_DATE_OUTPUT_FORMAT + ")");
     	// Each entry comes with an accompanying date and time.
-    	final String[] heightLineTokens = vitalResults.get(5).split(SPLIT_REGEX);
-    	final int feet = Integer.parseInt(heightLineTokens[2]);
+    	final List<String> heightLineTokens = Splitter.on(Pattern.compile(SPLIT_REGEX)).splitToList(vitalResults.get(5));
```

Comment:
You didn't change this either, but while we're at it... `SPLIT_REGEX` seems specific to vitals. (It's not used anywhere else.) You use other expressions in this file so I recommend either referring to vitals in the name or making it a local constant in this method.